src/generics.jl

```diff
@@ -28,6 +28,13 @@ Base.kron(A::AbstractPDMat, B::AbstractPDMat) = PDMat(kron(Matrix(A), Matrix(B))
 LinearAlgebra.isposdef(::AbstractPDMat) = true
 LinearAlgebra.ishermitian(::AbstractPDMat) = true
 
+"""
+    sqrt(A::AbstractPDMat)
+
+Return the matrix square root of `A`: the matrix `X` such that ``XX = A``
+"""
```

Comment:
This should not be documented according to the Julia documentation since it does not differ from the documentation for `sqrt(::AbstractMatrix)` (https://github.com/JuliaLang/julia/blob/dcc0efe26fd0490bf963be8d50dc6df83b91a868/stdlib/LinearAlgebra/src/dense.jl#L806) and `AbstractPDMat <: AbstractMatrix`.

```suggestion
# Generic fallback
```